mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+
+[mypy-pyramid.*]
+ignore_missing_imports = True
+[mypy-transaction]
+ignore_missing_imports = True
+[mypy-sqlalchemy.*]
+ignore_missing_imports = True
+[mypy-zope.*]
+ignore_missing_imports = True
```

Comment:
Correct, missing type hints( and no stubs). From my understanding this is how you are supposed to handle pyramid/zope.transaction/sqlachemy since they dont have type hints or stubs for mypy.

Reply:
> I think this is missing sqlalchemy[mypy] to get the type hints properly. ( As I don't think mypy --install-types adds that feature)
> 
> Details on sqlalchemy docs here:
> 
> https://docs.sqlalchemy.org/en/14/orm/extensions/mypy.html

sqlachemy does in fact have a mypy stub, removed the ignore for sqlalchemy.